wiki/osu!_wiki/Maintenance/en.md

```diff
@@ -0,0 +1,103 @@
+- `needs rebase`: the pull request has too many small unstructured commits, which need to be rebased and worded in a better way
+
+### Links and redirects
+
+Most osu! wiki articles have alternative links, which are set up using the [`redirect.yaml`](https://github.com/ppy/osu-wiki/blob/master/wiki/redirect.yaml) file. The redirects are meant to be used outside the osu! wiki, for example, on forums, or in the [chat](/wiki/Chat_Console), where they could be quickly turned into an inline reference:
+
+```
+According to [[RC]], this is forbidden.
+```
+
+When adding redirects for a new or existing article, keep in mind that they should be concise and designed for actual use.
+
+<!-- TODO: should probably briefly mention external links https://github.com/ppy/osu-web/issues/8086 and footnotes https://github.com/ppy/osu-wiki/issues/4911#issuecomment-893959588 once they are implemented -->
+
+### CI checks
+
+The osu! wiki repository uses [continuous integration](https://docs.github.com/en/actions/guides/about-continuous-integration) (CI) to automatically check incoming pull requests for different common errors. The list of checks is configured in the [`package.json`](https://github.com/ppy/osu-wiki/blob/master/package.json) file and includes several custom plugins written by the osu! wiki maintainers.
+
+The CI checks are run automatically on every commit of a recurring contributor. In order to have their pull requests merged, contributors are expected to fix errors reported by the CI. To see the [status of checks](img/ci-status.png), do the following:
+
+1. Scroll the pull request's page down, find the `osu-wiki continuous integration` status row, and click the `Details` link.
+2. On the new page, expand the `run remark on changed files` step. Each finding is accompanied by its exact location in a file and a short description of why it is an error.
+
+If you need help with decrypting the CI check errors, or fixing the issues, ask in the `#osu-wiki` channel on Discord.
+
+### Development
+
+osu! wiki is integrated into the osu! website, which means all technical feature requests should be [made and tracked](https://github.com/ppy/osu-web/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3Aarea%3Awiki) in the `ppy/osu-web` repository. To inform other contributors of your request, make sure to link the new issue in the `#osu-wiki` Discord channel, or osu! wiki's issue tracker.
+
+### Tools
+
+Some features are not directly related to the osu! website, but may be useful for contributing or general housekeeping. In this case, they are implemented by people capable of it and are **not** added to the website directly:
+
+- [osu-wiki status](https://clayton.cc/osu-wiki-status/en): list of articles per every language, and the category of maintenance they require (translation, update, stub expansion). Functionality requests: [ppy/osu-wiki#2486](https://github.com/ppy/osu-wiki/issues/2486)
```

Comment:
since these things are "inlined", this might be prettier

```suggestion
- [osu-wiki status](https://clayton.cc/osu-wiki-status/en): list of articles per every language, and the category of maintenance they require (translation, update, stub expansion). See [ppy/osu-wiki#2486](https://github.com/ppy/osu-wiki/issues/2486) for functionality requests.
```

Reply:
yes and also yes